src/Umbraco.Web.UI.NetCore/Umbraco.Web.UI.NetCore.csproj

```diff
@@ -4,35 +4,52 @@
         <TargetFramework>netcoreapp3.1</TargetFramework>
         <RootNamespace>Umbraco.Web.UI.NetCore</RootNamespace>
     </PropertyGroup>
-
     <ItemGroup>
+      <ProjectReference Include="..\Umbraco.Examine.Lucene\Umbraco.Examine.Lucene.csproj" />
+      <ProjectReference Include="..\Umbraco.Persistance.SqlCe\Umbraco.Persistance.SqlCe.csproj" />
+      <ProjectReference Include="..\Umbraco.PublishedCache.NuCache\Umbraco.PublishedCache.NuCache.csproj" />
       <ProjectReference Include="..\Umbraco.Web.BackOffice\Umbraco.Web.BackOffice.csproj" />
       <ProjectReference Include="..\Umbraco.Web.Common\Umbraco.Web.Common.csproj" />
       <ProjectReference Include="..\Umbraco.Web.Website\Umbraco.Web.Website.csproj" />
     </ItemGroup>
 
     <ItemGroup>
       <Folder Include="wwwroot\Media" />
+      <Folder Include="wwwroot\Umbraco\views\install" />
     </ItemGroup>
 
     <ItemGroup>
-      <Compile Remove="wwwroot\Umbraco\**" />
         <Compile Remove="App_Data\**" />
     </ItemGroup>
 
     <ItemGroup>
-      <EmbeddedResource Remove="wwwroot\Umbraco\**" />
         <EmbeddedResource Remove="App_Data\**" />
     </ItemGroup>
 
     <ItemGroup>
-      <None Remove="wwwroot\Umbraco\**" />
         <None Remove="App_Data\**" />
     </ItemGroup>
 
     <ItemGroup>
-        <Content Remove="wwwroot\Umbraco\**" />
         <Content Remove="App_Data\**" />
+        <Content Remove="wwwroot\Umbraco\views\install\machinekey.html" />
+        <Content Remove="wwwroot\Umbraco\views\install\permissionsreport.html" />
+        <Content Remove="wwwroot\Umbraco\views\install\starterkit.html" />
+        <Content Remove="wwwroot\Umbraco\views\install\upgrade.html" />
+        <Content Remove="wwwroot\Umbraco\views\install\user.html" />
+        <Content Remove="wwwroot\Umbraco\views\install\version7upgradereport.html" />
+        <Content Remove="wwwroot\Umbraco\views\install\continueinstall.html" />
+        <Content Remove="wwwroot\Umbraco\views\install\database.html" />
+        <Content Remove="wwwroot\Umbraco\views\install\error.html" />
+        <Content Update="Views\Install\Index.cshtml">
+          <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+          <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+        </Content>
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.3" />
+      <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.7.0" />
```

Comment:
why are we referencing `System.Configuration.ConfigurationManager` ?

Reply:
Currently a hack due to Lucene, can be removed when Examine is updated to netstandard